packages/x-tree-view/src/internals/plugins/useTreeViewFocus/useTreeViewFocus.types.ts

```diff
@@ -7,12 +7,14 @@ import { UseTreeViewExpansionSignature } from '../useTreeViewExpansion';
 
 export interface UseTreeViewFocusInstance {
   isNodeFocused: (nodeId: string) => boolean;
-  focusNode: (event: React.SyntheticEvent, nodeId: string | null) => void;
-  focusDefaultNode: (event: React.SyntheticEvent) => void;
-  focusRoot: () => void;
+  isTreeViewFocused: () => boolean;
+  canNodeBeTabbed: (nodeId: string) => boolean;
+  focusNode: (event: React.SyntheticEvent, nodeId: string) => void;
+  focusDefaultNode: (event: React.SyntheticEvent | null) => void;
+  removeFocusedNode: () => void;
 }
 export interface UseTreeViewFocusPublicAPI {
-  focusNode: (event: React.SyntheticEvent, nodeId: string | null) => void;
```

Comment:
@noraleonte with the introduction of `focusDefaultNode`, do we need to support `null` here?